docs/cli.mdx

```diff
@@ -36,9 +36,9 @@ Login with you Saleor Could account:
 saleor login
 ```
 
-The `saleor login` command allows you to establish the current user session in CLI. This will initiate an OAuth process. A browser window will open with the Saleor Cloud login page. Once authenticated, the CLI will receive an authentication token that will be stored locally for the CLI to access.
+The `Saleor login` command lets you establish the current CLI user sessio. This will initiate an OAuth process. A browser window will open with the Saleor Cloud login page. Once authenticated, the CLI will receive an authentication token that will be stored locally for the CLI to access.
```

Comment:
```suggestion
The `saleor login` command lets you establish the current CLI user session. This will initiate an OAuth process. A browser window will open with the Saleor Cloud login page. Once authenticated, the CLI will receive an authentication token that will be stored locally for the CLI to access.
```